crates/key-value-store/src/api/store.rs

```diff
@@ -0,0 +1,29 @@
+use async_trait::async_trait;
+use serde::{de::DeserializeOwned, Serialize};
+
+use crate::api::error::KeyValueError;
+
+#[async_trait]
+pub trait KeyValueStore<K: Clone + Sync + Send>: Clone + Sync + Send {
+    //
```

Comment:
Nit on style: remove the empty comment lines

Reply:
Followup, should it actually be triple-slash I use? That's what would go to docs, right?

Reply:
Done.